### PR TITLE
DAOS-15104 docs: Improve set-logmasks documentation in admin guide

### PR DESCRIPTION
**Pull request by tanabarr**
Provide details of different logging parameters that can be adjusted
with dmg server set-logmasks and describe reset behaviour.

Doc-only: true
Required-githooks: true

### Before requesting gatekeeper:

* [x] Two review approvals and any prior change requests have been resolved.
* [x] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [x] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [x] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [x] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate watchers.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

### CODE REVIEW

**Review by jolivier23 — APPROVED** [review]

File: docs/admin/administration.md [hunk 1]
```diff
@@ -63,38 +63,108 @@ severity, message, description, and cause.
...
 facilities to a given level.
 Settings will be applied to all running DAOS I/O Engines present in the configured dmg hostlist
-using the command `dmg server set-logmasks [-m <masks>]`.
-The command accepts named arguments for masks ('D_LOG_MASK'), streams ('DD_MASK') and subsystems
-('DD_SUBSYS).
-If no args are passed, then the log masks for each running engine will be reset to the values of
-engine "log\_mask" parameter and "env\_vars" 'DD_MASK' and 'DD_SUBSYS' assignments in the server
-config file (as set at the time of daos\_server startup).
-If a single arg is passed, then this will be used as the log masks setting.
+using the set-logmasks command.
```
Comment (knard-intel) [inline docs/admin/administration.md:75]: nit
```suggestion
using the `set-logmasks` command.
```

**Review by knard-intel — APPROVED** [review]
Very helpful and detailed documentation :)

File: docs/admin/administration.md [hunk 2]
```diff
@@ -63,38 +63,108 @@ severity, message, description, and cause.
...
+set-logmasks` without parameters would reset logging to config values and would be equivalent to the
+example given above.
 
-For usage of streams ('DD_MASK') and subsystems ('DD_SUBSYS') parameters, logging is described in
-detail in the
-[`Debugging System`](https://docs.daos.io/v2.6/admin/troubleshooting/#debugging-system)
-section.
+For more information on the usage of masks (`D_LOG_MASK`), streams (`DD_MASK`) and subsystems
+(`DD_SUBSYS`) parameters refer to the
+[`Debugging System`](https://docs.daos.io/v2.6/admin/troubleshooting/#debugging-system) section.
```
Comment (daltonbohning) [inline docs/admin/administration.md:167]: Existing, but
1. Should Debugging System be formatted like this? Maybe Just
```
[Debugging System]
```
2. Can these links not be dynamic? I.e. they are hardcoded to `docs.daos.io/v2.6`

**Review by daltonbohning — APPROVED** [review]

**Review by knard-intel — APPROVED** [review]

File: docs/admin/administration.md (hunk 2 above)
Comment (tanabarr) [reply docs/admin/administration.md:167]: yes to both, we should probably apply both points across the admin guide

**Pull request merged**